StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SpacingExtensions.cs

```diff
@@ -43,6 +44,71 @@ public static SyntaxTriviaList WithoutWhitespace(this SyntaxTriviaList syntaxTri
         }
 
         /// <summary>
+        /// Removes the leading and trailing trivia associated with a syntax node.
+        /// </summary>
+        /// <typeparam name="TNode">The type of syntax node to remove trivia from.</typeparam>
+        /// <param name="node">The syntax node to remove trivia from.</param>
+        /// <returns>A copy of the input syntax node with leading and trailing trivia removed.</returns>
+        /// <exception cref="ArgumentNullException">If <paramref name="node"/> is <see langword="null"/>.</exception>
+        public static TNode WithoutTrivia<TNode>(this TNode node)
+            where TNode : SyntaxNode
+        {
+            return node.WithoutLeadingTrivia().WithoutTrailingTrivia();
```

Comment:
It might be a good idea to check node for null here to get a consistend stack frame with the other extension methods.